Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "gbmu"
 version = "0.1.0"
-authors = ["Guilhem Smith <gsmith@student.42.fr>"]
+authors = [
```

Comment:
🙃

Reply:
😛  it was for testing, but send me your line for me to add it 👍